src/_repobee/git.py

```diff
@@ -69,7 +69,7 @@ def _ensure_repo_dir_exists(clone_spec: CloneSpec) -> None:
 
 
 def _git_init(dirpath):
-    captured_run(["git", "init"], cwd=str(dirpath))
+    subprocess.run("git init".split(), cwd=str(dirpath))
```

Comment:
I think you're missing something here. If the output is not captured, then it will go to stdout/stderr. That means that this function now causes the output of `git init` to be displayed to the user, which is not desirable.

Reply:
Oh, I thought the default behaviour would be to discard the output. In more technical terms, it would redirect the output to `/dev/null`. Since we do not care about the output here, should we do redirect the stdout and stderr to `/dev/null`?

Reply:
> Oh, I thought the default behaviour would be to discard the output.

A child process inherits the file descriptor table of its parent process (this is consistent across all operating systems and programming languages I know of), so if the parent process outputs to stdout and stderr, so will the child.

> Since we do not care about the output here, should we do redirect the stdout and stderr to /dev/null?

We could do that, but the benefit over just using `captured_run` isn't all too clear to me. Perhaps there's something different we can do here? Rename the function perhaps?

Reply:
I feel that using `subprocess.run` is very clear by itself given it is a common python function and we have passed very few arguments to it. If it needed a lot of extra arguments, a wrapper would be desirable in that case.

When I saw this function, my first thought was to reuse this everywhere where `subprocess.run` was used. However, it was definitely not needed because of the same arguments above. Moreover, for this specific case, we don't even want to capture output so a function name `captured_run` shouldn't be invoked in my opinion.

Reply:
> it's just an incredibly noisy function that takes way too many arguments.

In our case, we just pass four arguments and two of them are common - `stdout` and `stderr`. In my opinion, it was not enough for me to create a wrapper around it.

> This is why I suggested a rename of the function such that it's less confusing

It is not confusing at all. I wanted to use `captured_run` wherever we invoke `subprocess.run` for the sake of uniformity. In most of the places where the latter is invoked, the same set of arguments is provided. However, there were a lot of occurrences of `subprocess.run` and they were readable as well. Thus, I thought to remove `captured_run` instead to achieve the goal I thought of.

Reply:
> It is not confusing at all.

To some extent it has to be confusing, as in refactoring, you inadvertently changed the semantics of the program by letting the output go to stdout/stderr.

Anyway, there's a neater option available as of #986, because in Python 3.7+ there's a `capture_output` argument to `subprocess.run`. So we can just set `capture_output=True`, which would more conclusively obviate the need for a wrapper function.